.bazelrc

```diff
@@ -38,6 +38,7 @@ build --action_env=CC
 build --action_env=CXX
 build --action_env=LLVM_CONFIG
 build --action_env=PATH
+build --action_env=GOPROXY=https://goproxy.cn
```

Comment:
> MessageMetadata::clone() should handle header flags as well I think (used when mirroring requests).
> […](#)
> On Fri, Oct 29, 2021, 4:46 AM code ***@***.***> wrote: ***@***.**** commented on this pull request. Thanks 🌷 , looks great. One simple comment to start further review. ------------------------------ In .bazelrc <[#18806 (comment)](https://github.com/envoyproxy/envoy/pull/18806#discussion_r739043111)>: > @@ -38,6 +38,7 @@ build --action_env=CC build --action_env=CXX build --action_env=LLVM_CONFIG build --action_env=PATH +build --action_env=GOPROXY=https://goproxy.cn May be this is unnecessary for the community? — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <[#18806 (review)](https://github.com/envoyproxy/envoy/pull/18806#pullrequestreview-792772525)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAKJSA6E2AU5R5S7QDEPPF3UJJNOVANCNFSM5G3YAGKA> . Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

thx and done